src/Dfc.CourseDirectory.Web/Helpers/OnspdSearchHelper.cs

```diff
@@ -1,28 +1,30 @@
-﻿using Dfc.CourseDirectory.Models.Models.Onspd;
-using Dfc.CourseDirectory.Services.Interfaces.OnspdService;
-using Dfc.CourseDirectory.Services.OnspdService;
+﻿using System.Linq;
+using Dfc.CourseDirectory.Core.Search;
+using Dfc.CourseDirectory.Models.Models.Onspd;
+using Mapster;
 
 namespace Dfc.CourseDirectory.Web.Helpers
 {
     public class OnspdSearchHelper : IOnspdSearchHelper
     {
-        private readonly IOnspdService _onspdService;
+        private readonly ISearchClient<Core.Search.Models.Onspd> _searchClient;
 
-        public OnspdSearchHelper(IOnspdService onspdService)
+        public OnspdSearchHelper(ISearchClient<Core.Search.Models.Onspd> searchClient)
         {
-            _onspdService = onspdService;
+            _searchClient = searchClient;
         }
 
         public Onspd GetOnsPostcodeData(string postcode)
```

Comment:
This is only called 3 times from one class, you could trash this unnecessary abstraction completely?